### PR TITLE
Leaderboard fixes

### PR DESCRIPTION
**Pull request by NataliaMilgevskaya**
bug [Bug 90931](https://dev.azure.com/EvrosPowerPlatformTeam/Default/_workitems/edit/90931): Leaderboard does not respect the "timestamp" passed in fixed

[Enhancement 90887](https://dev.azure.com/EvrosPowerPlatformTeam/Default/_workitems/edit/90887): Align column titles

[Bug 90900](https://dev.azure.com/EvrosPowerPlatformTeam/Default/_workitems/edit/90900): Duplicate "Home" in navigation

### CODE REVIEW

**Comment by EdiWeeks** [comment]
@BenGWeeks, tested. All bugs and enhancements are resolved. This is ready to be merged.

File: tabs/src/types/global.d.ts [hunk 1]
```diff
@@ -68,3 +68,20 @@ declare module '*.png' {
   const content: any;
   export default content;
 }
+
```
Comment (BenGWeeks) [inline tabs/src/types/global.d.ts:71]: Nope - but these back into the local file. These don't need to be global. I moved them there for a reason :-)

**Review by BenGWeeks — CHANGES_REQUESTED** [review]

File: tabs/src/components/Leaderboard.tsx [hunk 2]
```diff
@@ -18,22 +18,6 @@ interface LeaderboardProps {
   timestamp?: number | null;
 }
 
-interface PrivateWalletTransaction {
```
Comment (BenGWeeks) [inline tabs/src/components/Leaderboard.tsx:21]: Would like this stuff back.

Comment (BenGWeeks) [reply tabs/src/components/Leaderboard.tsx]: I resolved this.

**Pull request merged**